source/rst/tax_smoothing_1.rst

```diff
@@ -501,44 +509,44 @@ spending process.
     plt.ylabel('Taxation')
     plt.show()
 
-Python class to solve Markov jump linear quadratic control problems
--------------------------------------------------------------------
+Python Class to Solve Markov Jump Linear Quadratic Control Problems
+===================================================================
 
 To implement the extension to the Barro model in which :math:`p_{t,t+1}`
 varies over time, we must allow the M matrix to be time-varying. From
 the mapping of the Barro model into the LQ framework, this means that
-our Q and W matrices will now also vary over time. We can solve such a
+our :math:`Q` and :math:`W` matrices will now also vary over time. We can solve such a
 model using the LQ_Markov class, which solves Markov jump linear
-quandratic control problems as described above.
+quandratic control problems as described above
 
 The code for the class can be viewed
-`here <https://github.com/QuantEcon/TaxSmoothing/blob/master/lq_markov.py>`__.
+`here <https://github.com/QuantEcon/TaxSmoothing/blob/master/lq_markov.py>`__
 
 The class takes a variable number of arguments, to allow for there to be
 an arbitrary :math:`N` states of the world. To accomodate this, the
 matrices for each state of the world must be held in a “namedtuple”. The
 value and policy functions are then found by iterating on the system of
 algebraic matrix Riccati equations. The solutions for :math:`P,F,\rho`
-are stored in Python “dictionaries”.
+are stored in Python “dictionaries”
 
 The class also contains a “method”, for simulating the model. This is an
 extension of a similar method in the LQ class, adapted to take into
 account the fact that the model’s matrices depend on the state of the
-world.
+world
 
 Below we import all functionality from this code.
 
 (You should download `the
 file <https://github.com/QuantEcon/TaxSmoothing/blob/master/lq_markov.py>`__
```

Comment:
Would you be able to implement this function (https://quanteconpy.readthedocs.io/en/latest/util/notebooks.html) and place the file in QuantEcon.Notebooks

Reply:
I will do